msgspec/_json_schema.py

```diff
@@ -142,6 +143,7 @@ def _get_doc(t: mi.Type) -> str:
     doc = getattr(cls, "__doc__", "")
     if not doc:
         return ""
+    doc = textwrap.dedent(doc)
```

Comment:
Should we also call `.strip()` here to strip leading/trailing whitespace?

```suggestion
    doc = textwrap.dedent(doc).strip()
```

I _think_ this makes sense. If so, you'll also want to update the test below to match.

Reply:
I think `strip()` is too much: If the docstring starts with an indented code block, like this:

```
"""
    some code

regular paragraph.
"""
```
it would be turned into `"some code\n\nregular paragraph"`.

It should be save to remove just the leading and trailing newlines, though.

Reply:
I've pushed a commit that strips newlines and updated the test.